yaml/pois.yaml

```diff
@@ -414,3 +414,45 @@ filters:
     min_zoom: 15
     output:
       kind: {col: tags->whitewater}
+  - filter: {aerialway: pylon}
```

Comment:
It seems like we need some criteria about which layer something goes into? Otherwise, we might as well just have `points`, `lines`` and `polygons` layers. Why is, for example, `landuse` separate from `buildings` or `earth`, and are there similar categories to break down `pois` into?